src/bacnet_server/bac_server.py

```diff
@@ -193,6 +193,7 @@ def remove_all_points(self):
 
     def _ip_address(self, bacnet_server):
         use_nic = self.config.enable_ip_by_nic_name
+        #TODO @binod call the db and use the  enable_ip_by_nic_name, and ip_by_nic_name as this means we dont need the config file
```

Comment:
@RaiBnod can you please add in the logic to get this from the DB so we dont use the config file